### PR TITLE
add: patch and delete endpoint for business account

### PR DESCRIPTION
**Pull request by yusuke99**

### CODE REVIEW

File: server/controllers/businesses.js [hunk 1]
```diff
@@ -109,6 +109,53 @@ router.post("/", async (req, res) => {
...
+
+  const updateInfo = {};
+  if (req.body.name) updateInfo["name"] = req.body.name;
+  if (req.body.last_name) updateInfo["address_street"] = req.body.address_street;
+  if (req.body.email) updateInfo["address_city"] = req.body.address_city;
+  if (req.body.phone) updateInfo["address_zip"] = req.body.address_zip;
+  if (req.body.phone) updateInfo["phone"] = req.body.phone;
+  if (req.body.phone) updateInfo["business_type"] = req.body.business_type;
+  if (req.body.phone) updateInfo["capacity"] = req.body.capacity;
+  if (req.body.phone) updateInfo["price"] = req.body.price;
```
Comment (catboop) [inline server/controllers/businesses.js]: these are all checking `req.body.phone`

Comment (catboop) [inline server/controllers/businesses.js]: do you need this logic? isn't this going to be the same thing as the `req.body`?

File: server/controllers/businesses.js [hunk 2]
```diff
@@ -109,6 +109,53 @@ router.post("/", async (req, res) => {
...
+
+  await db
+  .select("*")
+  .table("businesses")
+  .where({ user_id })
+  .update(updateInfo);
+
+  res.send("Business information is updated");
```
Comment (catboop) [inline server/controllers/businesses.js]: i changed my mind. maybe let's just send back the updated biz deets:
```suggestion
  const updatedBiz = await db
  .select("*")
  .table("businesses")
  .where({ user_id })
  .update(updateInfo);

  res.send(updatedBiz);
```

**Review by catboop — CHANGES_REQUESTED** [review]

**Comment by yusuke99** [comment]
@catboop Also updated patch and delete for users

**Review by catboop — APPROVED** [review]

**Pull request merged**